src/core/mac/mac.cpp

```diff
@@ -1343,6 +1344,7 @@ void Mac::ReceiveDoneTask(Frame *aFrame, ThreadError aError)
         {
             for (Receiver *receiver = mReceiveHead; receiver; receiver = receiver->mNext)
             {
+                otDumpDebgMac("RX", aFrame->GetHeader(), aFrame->GetLength());
```

Comment:
I would suggest to add another packet dump option strictly on the data path, thus allowing folks to either poke across all packets (here) or just poke on the data path.